cmd/rbac/rbac_create_subject.go

```diff
@@ -0,0 +1,64 @@
+package rbac
+
+import (
+	"github.com/spf13/cobra"
+	"gitlab.com/stackvista/stackstate-cli2/generated/stackstate_api"
+	"gitlab.com/stackvista/stackstate-cli2/internal/common"
+	"gitlab.com/stackvista/stackstate-cli2/internal/di"
+)
+
+const STQLVersion = "0.0.1"
+
+type CreateSubjectArgs struct {
+	Subject string
+	Scope   string
+}
+
+func CreateSubjectCommand(deps *di.Deps) *cobra.Command {
+	const ScopeUsage = "The query in STQL that will be prepended to every topology element retrieved in StackState. " +
```

Comment:
Maybe move this to `flags.go` too? The `ScopeUsage` and `Scope` flag are there too.